src/bookshelf/rebar.lock

```diff
@@ -37,8 +43,16 @@
   {git,"https://github.com/chef/erlware_commons",
        {ref,"f511ed875bea87212ba9ee40ade1c9f53498ce79"}},
   0},
- {<<"fs">>,{pkg,<<"fs">>,<<"6.1.1">>},1},
- {<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
+% {<<"fs">>,{pkg,<<"fs">>,<<"6.1.1">>},1},
+ {<<"fs">>,
```

Comment:
is this here because of license_scout issues? Is license scout making it so that we cannot pull the hex packages as erlang dependencies?

Reply:
Do we know how this worked with the previous versions of the dependencies?

Reply:
@lbakerchef -  I feel, We need to understand why the license_scout behaviour is different in this case ?

Reply:
@PrajaktaPurohit asked me to create an issue about it.  It's in the backlog now.

Reply:
license_scout changes have been removed from this PR, and put in the final PR for the Erlang upgrade.